src/get-started/java-binding.md

```diff
@@ -104,6 +102,21 @@ blockchain state hash, which is included in each committed block. When using
 implements [`Schema`][schema] interface; when implementing
 `Service` directly – in the service itself.
 
+### Serialization
+
+Exonum stores an information as a byte arrays. That's because serialization takes the place.
```

Comment:
Exonum stores information as byte arrays. That's because serialization takes place.
Perhaps, this could be rephrased into a single sentence, for example:
Due to serialization, Exonum stores information as byte arrays.